Makefile

```diff
@@ -0,0 +1,70 @@
+.PHONY: help test test-net48 test-net8 test-net9 lint fmt check
+
+# Default target when running just 'make'
+help:
+	@echo "Available targets:"
+	@echo "  test       - Run all tests"
+	@echo "  test-net48 - Run tests for .NET Framework 4.8"
+	@echo "  test-net8  - Run tests for .NET 8.0"
+	@echo "  test-net9  - Run tests for .NET 9.0"
+	@echo "  lint       - Verify code formatting and style"
+	@echo "  fmt        - Apply code formatting"
+	@echo "  check      - Run all checks (lint + test)"
+
+# Run tests for all supported frameworks
+test: test-net48 test-net8 test-net9
+	@echo "✅ All tests completed successfully!"
+
+# Restore NuGet packages
+restore:
+	@echo "📦 Restoring NuGet packages..."
+	@dotnet restore ./OpenFga.Sdk.sln
+
+# Run tests for .NET Framework 4.8
+test-net48: restore
+	@echo "🚀 Running tests for .NET Framework 4.8..."
+	@dotnet test --framework net48 --no-build
+
+# Run tests for .NET 8.0
+test-net8: restore
+	@echo "🚀 Running tests for .NET 8.0..."
+	@dotnet test --framework net8.0 --no-build
```

Comment:
The --no-build flag skips building, but the dependency is only restore, not a build target. Tests will fail because there's no compiled output.